CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+################################################################################
+# Copyright (c) 2018, 2018 IBM Corp. and others
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+# or the Apache License, Version 2.0 which accompanies this distribution and
+# is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following
+# Secondary Licenses when the conditions for such availability set
+# forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+# General Public License, version 2 with the GNU Classpath
+# Exception [1] and GNU General Public License, version 2 with the
+# OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] http://openjdk.java.net/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+################################################################################
+
+cmake_minimum_required(VERSION 3.4 FATAL_ERROR)
+
+project(openj9)
+include(FindJava)
```

Comment:
What does `FindJava` do?  Is it something we authored?  

The root of my concern is that we want to be able to specify the Java being used to ensure it's the same bootjdk used by the rest of the build.

Reply:
It is a built in module from cmake. you can point it to a specific java by setting `JAVA_HOME`
https://cmake.org/cmake/help/v3.4/module/FindJava.html

Reply:
I think this should use the bootjdk as configured in the extension project rather than the default for the build machine.